### DISCUSSION TITLE
drag_drop plugin without JQuery?

### DISCUSSION THREAD

**Discussion by jrochkind** [orchidjs/tom-select#69] [Ideas]
The drag_drop plugin still requires not only JQuery, but the pretty heavy-weight  jQuery UI "sortable" module. 

I think the drag_drop plugin could probably be implemented fairly straightforwardly with [HTML5 drag-and-drop API](https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API) instead.  It could even be an alternate, `drag_drop_html5` or what have you. 

I wonder if there's any interest, or anyone who feels like that implementing that?

**Comment by ghost** [comment]
View my quick&dirty example: https://github.com/orchidjs/tom-select/discussions/576

I need a little more experience with tom-select, then i can write a d&d plugin for it. The current solution works for the moment.
